client/lib/directly/index.js

```diff
@@ -110,6 +103,30 @@ export function initialize() {
 			}
 		} );
 	} );
+}
+
+/**
+ * Initializes the RTM widget if it hasn't already been initialized. This sets up global
+ * objects and DOM elements and requests the vendor script.
+ *
+ * @returns {Promise} Promise that resolves after initialization completes or fails
+ */
+export function initialize() {
+	if ( directlyPromise instanceof Promise ) {
+		return directlyPromise;
```

Comment:
Yep, you're totally right. I know you mentioned it here too: https://github.com/Automattic/wp-calypso/pull/11871#discussion_r105221703 This is a shortcoming of the early implementation of the library (note: this is not new code, despite the diff making it appear that it is).

Do you see this as a blocker for the initial launch of Directly? I've got it documented on the project's "future improvements" list. I'd prefer to push past to hit deadlines, given that this will be a slow experimental rollout and the worst case scenario is "nothing has changed from the way things were before" (the user sees the Forums support form). If we decide to commit longer-term to Directly there's a handful of other cleanup tasks we'll want to triage as well.